guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
+| `HOLOSCAN_OUTPUT_PATH`       | `/var/holoscan/output/`    | Folder Path | Path to the output folder for the Application.                        |
+| `HOLOSCAN_WORKDIR`           | `/var/holoscan/ `          | Folder Path | Path to the Application's working directory.                          |
+| `HOLOSCAN_MODEL_PATH`        | `/opt/holoscan/models/`    | Folder Path | Path to the Application's models directory.                           |
+| `HOLOSCAN_CONFIG_PATH`       | `/var/holoscan/app.yaml`   | File Path   | Path to the Application’s configuration file.                         |
+| `HOLOSCAN_APP_MANIFEST_PATH` | `/etc/holoscan/app.config` | File Path   | Path to the Application’s configuration file.                         |
+| `HOLOSCAN_PKG_MANIFEST_PATH` | `/etc/holoscan/pkg.config` | File Path   | Path to the Application’s configuration file.                         |
+| `HOLOSCAN_DOCS`              | `/opt/holoscan/docs`       | Folder Path | Path to the folder containing application documentation and licenses. |
 
-A MAP MUST be in a format which supports local execution in a development environment.
+# Architecture & Design
 
-> [Note]
-> See [MONAI Operating Environments](monai-operating-environments.md) for additional information about supported environments.
+## Description
 
+The MONAI Application Package (MAP) is a functional package designed to act on datasets of a prescribed format. A MAP is a container image that adheres to the specification provided in this document.
 
-### Containerization
+## Application
 
-A MAP SHALL be a containerized application to maximize portability of its application.
+The primary component of a MAP is the application. The application is provided by an application developer and incorporated into the MAP using the MONAI Deploy Application Packager.
 
+All application code and binaries SHALL be in the `/opt/holoscan/app/` folder, except for any dependencies installed by the MONAI Deploy Packager during the creation of the MAP.
 
-### Compatible with Kubernetes
+All AI models (PyTorch, TensorFlow, TensorRT, etc.) SHOULD be in separate sub-folders of the `/opt/holoscan/models/` folder. In specific use cases where the app package developer is prevented from enclosing the model files in the package/container due to intellectual property concerns, the models can be supplied from the host system when the app package is run, e.g., via the volume mount mappings and the use of container env variables.
 
-A MAP SHALL NOT be in a format which inhibits or hampers the ability to deploy it using Kubernetes.
+## Manifests
 
+A MAP SHALL contain two manifests: the application manifest and the package manifest. The package manifest shall be stored in `/etc/holoscan/pkg.json`, and the application manifest shall be stored in `/etc/holoscan/app.json`. Once a MAP is created, its manifests are immutable.
```

Comment:
`immutable` -> `expected to be immutable`

Adding back the words `expected to be` from the previous version, because the mechanism to ensure they are `immutable` is not clear nor specified.